apps/epic-web/src/pages/index.tsx

```diff
@@ -63,8 +63,16 @@ const Index: NextPage<{
   const {data: commerceProps, status: commercePropsStatus} =
     trpc.pricing.propsForCommerce.useQuery({
       ...router.query,
-      productId: product.productId,
+      productId:
+        router.query.code || router.query.coupon
+          ? undefined
+          : product.productId,
```

Comment:
`product.productId` is the hard coded default site product, but if there is a coupon or code in the url we don't want to use that

potential problem: coupon/code is general discount and not restricted to a product. what happens?